sklearn/tests/test_pipeline.py

```diff
@@ -961,36 +979,44 @@ def test_set_feature_union_step_drop():
 
 
 def test_step_name_validation():
+    error_message_1 = r"Estimator names must not contain __: got \['a__q'\]"
```

Comment:
Please also use `re.escape` for this one instead of manual escaping, for consistency.